flang/lib/Lower/Bridge.cpp

```diff
@@ -1926,8 +1926,23 @@ class FirConverter : public Fortran::lower::AbstractConverter {
     return builder->createIntegerConstant(loc, controlType, 1); // step
   }
 
+  void addLoopAnnotationAttr(IncrementLoopInfo &info) {
+    mlir::BoolAttr f = mlir::BoolAttr::get(builder->getContext(), false);
+    mlir::LLVM::LoopVectorizeAttr va = mlir::LLVM::LoopVectorizeAttr::get(
+        builder->getContext(), /*disable=*/f, {}, {}, {}, {}, {}, {});
+    // Create distinct access group
+    mlir::LLVM::AccessGroupAttr ag =
+        mlir::LLVM::AccessGroupAttr::get(builder->getContext());
```

Comment:
We probably do not need the access group for this since this involves dependency management and you have specifically said that the loop should be vectorizable.

Reply:
I added the access group to match the attributes that classic flang and clang add to the loops. I can remove it if you don't think it's  necessary.

Reply:
Please remove the access groups unless you have confirmed that it is required.